src/_guides/libraries/private-files.md

```diff
@@ -40,8 +37,10 @@ doc/api/
 {% endprettify %}
 
 **Don't commit** files and directories
-dropped by other development environments.
-For example:
+created by other development environments.
+For example, if your development environment creates
+any of the following files,
+consider putting them in a global ignore file:
```

Comment:
global *git* ignore file

Reply:
I was trying not to be Git-centric.

Reply:
What other SCM global ignore files did you have in mind? 😄

Reply:
They exist. :) SVN, CVS, ... please don't make me remember more.